docs/markdown/Getting Help/the-pants-community/team.md

```diff
@@ -43,18 +43,19 @@ Pants open source project has had many team members in over a decade. The curren
-| ---------------------- | ------------------------------------------------------------------------------------------------------- |
-| **Chris Williams**     |                                                                                                         |
-| **Doron Somech**       | "Improving Scala dependency inference to support our codebase"                                          |
-| **Jacob Floyd**        | Gave the idea for `skip_flake8` et al, which grew into a flagship feature: incremental adoption         |
-| **Jonas Stendahl**     | "Adding formatting and linting support for Protobuf"                                                    |
-| **Marcelo Trylesinski**            | "Improving onboarding experience"                                                           |
-| **Nick Grisafi**       | "Participating in podcasts with maintainers (Eric and Josh) on developer experience and Pants!"         |
-| **Raúl Cuza**          | "The first time I was able to help someone else with pants on the Slack <https://pantsbuild.slack.com>." |
-| **Rhys Madigan**       | "Making unit test run times easier to understand"                                                       |
-| **Tansy Arron-Walker** | "Writing my first task in the new engine was really exciting!"                                          |
-| **Thales Menato**      | "Making sure Pants can run even under heavily secured work environments!"                               |
+| **NAME**                | **PROUDEST CONTRIBUTION**                                                                                |
+|-------------------------|----------------------------------------------------------------------------------------------------------|
+| **Chris Williams**      |                                                                                                          |
+| **Doron Somech**        | "Improving Scala dependency inference to support our codebase"                                           |
+| **Jacob Floyd**         | Gave the idea for `skip_flake8` et al, which grew into a flagship feature: incremental adoption          |
+| **Jonas Stendahl**      | "Adding formatting and linting support for Protobuf"                                                     |
+| **Marcelo Trylesinski** | "Improving onboarding experience"                                                                        |
+| **Nick Grisafi**        | "Participating in podcasts with maintainers (Eric and Josh) on developer experience and Pants!"          |
+| **Raúl Cuza**           | "The first time I was able to help someone else with pants on the Slack <https://pantsbuild.slack.com>." |
+| **Rhys Madigan**        | "Making unit test run times easier to understand"                                                        |
+| **Shantanu Kumar**      |                                                                                                          |
```

Comment:
All the diff except this line is my editor reformatting the table to align properly in source.